packages/react-google-maps-api/src/LoadScript.tsx

```diff
@@ -40,6 +32,7 @@ const DefaultLoadingElement = () => (
 
 class LoadScript extends React.PureComponent<LoadScriptProps, LoadScriptState> {
   public static defaultProps = {
+    id: 'script-loader',
     libraries: [] // Do not remove!,
```

Comment:
We should add ```version: "weekly"``` since this is the version used by default

Reply:
I had there for a brief moment, but since it's default within API, I decided to remove it. If someone wants to be explicit, they can specify the version.